gradle-scripts/docker.gradle

```diff
@@ -2,7 +2,7 @@ docker {
     javaApplication {
         baseImage = 'java:openjdk-8-jre-alpine'
         maintainer = 'Hesham Massoud [hesham.massoud@commercetools.com]'
-        tag = 'commercetools/commercetools-project-sync:' + version
+        images = ['commercetools/commercetools-project-sync:' + version]
```

Comment:
what is the reason for that change ? any link or documentation would be helpful.

Reply:
@ahmetoz https://bmuschko.github.io/gradle-docker-plugin/#change_log
See v6.0.0

Reply:
I am not sure from that change why not image ? instead of a plural image(s)

Reply:
OK, from the example it looks like we need to use images instead of tags 😉 
```
docker {
    javaApplication {
        baseImage = 'dockerfile/java:openjdk-7-jre'
        maintainer = 'Benjamin Muschko "benjamin.muschko@gmail.com"'
        ports = [9090, 5701]
        images = ['jettyapp:1.115', 'jettyapp:latest']
        jvmArgs = ['-Xms256m', '-Xmx2048m']
    }
}
```